### DISCUSSION TITLE
ffmpeg streaming to AzuraCast: Seems not to support moint point /

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6131] [Q&A] [answered]
I’m trying to stream to my DJ input using ffmpeg n5.1.2 but it seems ffmpeg cannot handle the mount point `/` and wants a mount name. This is what I tried:

```bash
ffmpeg -i test.mp3 \
  -c:a libmp3lame -b:a 128k \
  -f mp3 -content_type 'audio/mpeg' \
  icecast://moonbase:123456@azura.local:8005/
```

Response from ffmpeg:

```
[icecast @ 0x5588001ca9c0] No mountpoint (path) specified!
icecast://moonbase:123456@azura.local:8005/: Input/output error
```

Q1. Anyone got this to work?

Q2: Do we have a means to set a name for the mount, like `/source`, `/dj` or such?

**Comment by scysys** [comment]
You can try to set ; at the end of the url.

Normaly

http://example.com:8000/stream

Try

http://example.com:8000/;

**Reply by Moonbase59** [reply]
Nope, same error. Above would normally be used for ShoutCast servers, to disable browser detection.

**Reply by scysys** [reply]
Was just an idea :) Tried with the mountpoint in the URL?

```
ffmpeg -i test.mp3 \
  -c:a libmp3lame -b:a 128k \
  -f mp3 -content_type 'audio/mpeg' \
  icecast://moonbase:123456@azura.local:8005/stream
```

**Reply by Moonbase59** [reply]
That’s exactly the culprit: There is only the mountpoint `/` for the DJ input, and ffmpeg seems not able to handle that.

**Reply by scysys** [reply]
just saw the answer. one more idea

```
ffmpeg -i test.mp3 \
  -c:a libmp3lame -b:a 128k \
  -f mp3 -content_type 'audio/mpeg' \
  icecast://moonbase:123456@azura.local:8005/
  /
```

**Comment by Moonbase59** [comment, marked as answer]
Ok, got it working. Easier than I thought.

1. Enable Advanced Settings.
2. Under _(Station) → Edit Profile → Streamers/DJs → Customize DJ/Streamer Mount Point_, set the name of the mount point. (Defaults to `/`, I set it to `/live`).
3. Restart Broadcasting.
4. Ffmpeg can now stream to your DJ mount. It needs the `-re` option _before_ the input file, so it uses the native frame rate for streaming:
```bash
ffmpeg -re -i test.mp3 \
  -c:a libmp3lame -b:a 128k -ar 44100 -ac 2 \
  -f mp3 -content_type 'audio/mpeg' \
  icecast://moonbase:123456@azura.local:8005/live
```
5. In case you use the same mount for other clients like _ices0_, _Mixxx_ or _IDJC_, don’t forget to change the mount from `/` to `/live` in there, too!

---

You can even convert FLAC and restream these as MP3 into your station. For this, you _must_ exclude any (cover) image streams using `-vn`, and use a `-vsync cfr`:

```bash
ffmpeg -vsync cfr -re -i test.flac \
  -c:a libmp3lame -b:a 128k -ar 44100 -ac 2 \
  -vn \
  -f mp3 -content_type 'audio/mpeg' \
  icecast://moonbase:123456@azura.local:8005/live
```
